radixdlt/src/main/java/com/radixdlt/consensus/VertexStore.java

```diff
@@ -69,10 +70,14 @@ public VertexStore(
 		this.engine = Objects.requireNonNull(engine);
 		this.counters = Objects.requireNonNull(counters);
 		this.highestQC = Objects.requireNonNull(rootQC);
-		try {
-			this.engine.checkAndStore(genesisVertex.getAtom());
-		} catch (RadixEngineException e) {
-			throw new IllegalStateException("Could not store genesis atom: " + genesisVertex.getAtom(), e);
+
+		if (genesisVertex.getAtom() != null) {
```

Comment:
Do we allow the genesis vertex not to have an atom now?

Reply:
yes should be allowable